src/lbcamden/components/search/template.njk

```diff
@@ -2,15 +2,50 @@
   {% set classNames = classNames + " " + params.classes %}
 {% endif %}
 
-<form class="lbcamden-search {{ classNames }} {% if params.alt %} lbcamden-search--alt{% endif %}" action="/"
-      method="post" autocomplete="off" role="search">
-  <label class="govuk-label govuk-label--l govuk-visually-hidden" for="lbcamden-search__box">
-    {{ params.labelText | default('Search Camden.gov.uk') }}
+{%- if params.action %} 
+  {% set action = params.action %}
```

Comment:
@oluoluoxenfree As discussed can you tweak the indentation of these if statements to bring the conditional tags inline with each other.